docs/SupportedCommands/Commands.md

```diff
@@ -74,20 +74,31 @@ Examples:
 |----------------------------|-------------------------------|------------------------------------------|
 | `className`                | 					string	 				     | 					4.5.1+	 				|
 | `compositeOpacity`         | 					float	 				       | 					4.2.14+	 				|
+| `hasOpacity                | 					bool	 				       | 					4.5.1+	 				|
 | `hasFocus`                 | 					bool	 				       | 					4.5.1+	 				|
 | `id`                       | 					string			         | 					4.5.1+	 				|
 | `isDisplayed`              | 					bool	 				       | 					4.5.1+	 				|
 | `isHittable`               | 					bool	 				       | 					4.5.1+	 				|
 | `localOpacity`             | 					float	 				       | 					4.2.14+	 				|
 | `name`                     | 					string	 				     | 					4.2.1+	 				|
 | `text`                     | 					string	 				     | 					4.5.1+	 				|
+| `visibilityFlag`           | 					bool	 				       | 					4.5.1+	 				|
```

Comment:
Do we want to follow the same convention for bools here, i.e. `isVisible`?

Reply:
I used `visibilityFlag` to avoid confusion. As the visibility is not garantied if the flag is set.